A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -271,20 +287,24 @@ class FMCMainDisplay extends BaseAirliners {
         }
         SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex);
     }
+
     get cruiseFlightLevel() {
         return this._cruiseFlightLevel;
     }
+
     set cruiseFlightLevel(fl) {
-        this._cruiseFlightLevel = Math.round(fl / 5) * 5;
+        this._cruiseFlightLevel = fl;
         SimVar.SetSimVarValue("L:AIRLINER_CRUISE_ALTITUDE", "number", this._cruiseFlightLevel * 100);
     }
```

Comment:
Relevant lines, sorry for the mess 😆

Reply:
no round on decimals?

Reply:
you need a conditional check to ensure the pilot hasn't entered any decimals so on the proceeding `setCruiseFlightLevelAndTemperature()` put a conditon like `if (fl === Math.round(fl))` then if true then assign the cruise level otherwise print the error message

Reply:
> you need a conditional check to ensure the pilot hasn't entered any decimals so on the proceeding setCruiseFlightLevelAndTemperature() put a conditon like if (fl === Math.round(fl)) then if true then assign the cruise level otherwise print the error message

I did not change anything other than the `round` to 5, error message is unrelated to this PR. I can re-add the `round` without rounding to 5 though.

Reply:
yeah I'd add back the round then